scripts/wrappers/security-init.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+set -eux
+
+
+usage() {
+cat << EOF
+usage: start.sh --init-security yes --admin-password ...
+To be ran / setup once per cluster - or when wanting to rebuild the security index.
+--admin-password  (Optional)    Passphrase of the admin key
+--help                          Shows help menu
+EOF
+}
+
+
+# Args
+admin_password=""
+
+
+# Args handling
+function parse_args () {
+    # init-security boolean - from the charm, this should be based on a flag on the app data bag.
+    local LONG_OPTS_LIST=(
+        "admin-password"
```

Comment:
while this is useful for testing, I would encourage picking up passwords from an environment variable, vault/keyring or a file; this will be recorded in your history and even if not, visible when typing